DevelopersGuide.adoc

```diff
@@ -1708,7 +1708,7 @@ Our mutation can now become:
   [{list-id   :list/id
     person-id :person/id}]
   (action [{:keys [state]}]
-    (swap! state merge/remove-ident* [:person/id person-id] [:list/id list-id :list/people])))
```

Comment:
Your change is incorrect...the original code is correct.  Did you try this??? I would be amazed if that does anything useful.

Reply:
Thanks for trying to help, though :)   I appreciate it.

Reply:
Thank you for this great work, loving Fulcro so far :)

But there is something I don't understand about your comment:
* In application's current state, Fulcro Inspector shows a map with keys :enemies and :friends
![image](https://user-images.githubusercontent.com/1485192/72749820-5a696080-3bbb-11ea-9978-81aa424d56d5.png)

* With your path, whenever I click the delete button, the item doesn't dissapear but after transacting the mutation there is some new element empty element
![image](https://user-images.githubusercontent.com/1485192/72750320-d87a3700-3bbc-11ea-8316-43295a7a649f.png)

When I first saw this, after looking remote-ident* docs thought that the problem was with the route, so I deleted the first element, and the program worked just fine after the change.

What am I doing / understanding wrong?

Thanks in advance!

Reply:
The app state you are showing me is wrong. Possilby bad copy into your app.  It looks like you are missing `:ident` on the List component.  Please use Clojurians #fulcro slack for support/help.